components/DonatedItemsList.tsx

```diff
@@ -34,14 +38,22 @@ const DonatedItemsList: React.FC<DisplayDonatedItemsProps> = ({ userId }) => {
   }, []);
 
   const handleDeleteSuccess = (deleteItemId: number) => {
+    const deletedItem = storeItems.find((item) => item.id === deleteItemId);
+    setDeletedItemName(deletedItem?.item_name || 'Unknown name');
```

Comment:
stop giving things a default value

- we need to let the code break if something is wrong, otherwise we're hiding our future bugs
- TypeScript is designed to pester you, if it's not happy with your types you need to be clearer about what you want returned - _not to give if a fallback_